source/Planet.cpp

```diff
@@ -472,10 +472,22 @@ void Planet::Bribe(bool fullAccess) const
 // Demand tribute, and get the planet's response.
 string Planet::DemandTribute(PlayerInfo &player) const
 {
-	if(player.GetCondition("tribute: " + name))
-		return "We are already paying you as much as we can afford.";
+	static const int TRIBUTE_IGNORED = 1;
```

Comment:
They are not used by any other function in Planet.cpp, so I would say they are fine where they are. They will get set in the first call, and any subsequent calls to DemandTribute() will reuse them without re-setting them.